### PR TITLE
Change appointments year_of_birth to date_of_birth

### PR DESCRIPTION
**Pull request by AndrewVos**

### CODE REVIEW

**Comment by benbarnett** [comment]
How does this render? Three dropdowns?

**Comment by AndrewVos** [comment]
@benbarnett Kind of. It's just the html5 picker. Attached screenshot ^^

File: db/migrate/20161012143328_add_date_of_birth_to_appointments.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddDateOfBirthToAppointments < ActiveRecord::Migration[5.0]
+  def change
+    add_column :appointments, :date_of_birth, :datetime
```
Comment (dwhenry) [inline db/migrate/20161012143328_add_date_of_birth_to_appointments.rb]: why are we storing DOB as a datetime?

Comment (AndrewVos) [inline db/migrate/20161012143328_add_date_of_birth_to_appointments.rb]: Wow good catch! Sorted.

File: spec/features/contact_centre_agent_creates_appointments_spec.rb [hunk 2]
```diff
@@ -61,7 +61,7 @@ def then_that_appointment_is_stored
     expect(appointment.email).to eq 'email@example.org'
     expect(appointment.phone).to eq '0000000'
     expect(appointment.mobile).to eq '1111111'
-    expect(appointment.year_of_birth).to eq '1891'
+    expect(appointment.date_of_birth).to eq '1983-10-23 00:00:00.000000000 +0000'
```
Comment (Guntrisoft) [inline spec/features/contact_centre_agent_creates_appointments_spec.rb]: Should this still be in this format now it's a date?

Comment (AndrewVos) [inline spec/features/contact_centre_agent_creates_appointments_spec.rb]: Yup yup was just fixing that :)

**Pull request merged**